SPECS/haproxy/haproxy.spec

```
@@ -30,6 +31,7 @@ Requires:       %{name} = %{version}-%{release}
 
 %prep
 %setup -q
+%patch 0 -p1
```

Can we please switch to autosetup and drop `%setup`. Something like:

```suggestion
%autosetup -p1
```

---

Done.